README.md

```diff
@@ -50,8 +50,6 @@ Attributes Reference
 
 The following attributes are exported:
 
-* `id` - Amazon Resource Name (ARN) of the secret.
```

Comment:
@kwessel, I removed this because it provides an identical value as `arn` and didn't see the value of producing two outputs with the same value.